app/src/main/res/values-nl/help.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="help_create_new"><![CDATA[
+        <h1>Portemonnee maken - Nieuw</h1>
+        <p>Als je een nieuw Monero-adres nodig hebt.</p>
+        <p>Voer een unieke naam en wachtwoord in.
+        Met het wachtwoord worden de gegevens in je portemonnee op dit apparaat beveiligd. Gebruik een sterk wachtwoord - of liever een wachtzin van meerdere woorden.</p>
+        <h2>Schrijf je hersteltekst op!</h2>
+        <p>Op het volgende scherm wordt de hersteltekst (\"mnemonic seed\") van 25 woorden getoond.
+        Dit is het enige dat je nodig hebt om later je portemonnee te herstellen en volledige toegang tot je geld te krijgen.
+        Het is belangrijk om deze gegevens geheim te houden, want hiermee kan <em>iedereen</em> volledige toegang tot je geld krijgen!</p>
```

Comment:
Misschien 'verborgen' ipv 'geheim'.

Reply:
Ja, dat kan. Misschien zelfs "veilig en verborgen te bewaren"? Het is de vertaling van "Keeping this secure and private"

Reply:
> Het is belangrijk om deze gegevens verborgen te houden, want hiermee kan men volledige toegang tot je portemonnee krijgen met alle gevolgen van dien.</p>

oid. : P

Reply:
Nice!

Reply:
I updated this part locally and on Github. @skftn Can you give an opinion about whether the translation as a whole is good enough to publish? That's the main purpose of reviewing :)